ZAPD/OtherStructs/CutsceneMM_EnumData.h

```diff
@@ -0,0 +1,200 @@
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_530), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_531),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_532), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_533),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_534), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_535),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_536), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_537),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_538), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_539),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_540), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_541),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_542), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_543),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_544), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_545),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_546), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_547),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_548), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_549),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_550), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_551),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_552), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_553),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_554), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_555),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_556), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_557),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_558), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_559),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_560), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_561),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_562), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_563),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_564), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_565),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_566), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_567),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_568), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_569),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_570), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_571),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_572), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_573),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_574), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_575),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_576), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_577),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_578), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_579),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_580), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_581),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_582), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_583),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_584), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_585),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_586), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_587),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_588), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_589),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_590), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_591),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_592), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_593),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_594), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_595),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_596), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_597),
+	ENUM_TO_STRING(CS_CMD_ACTOR_CUE_598), ENUM_TO_STRING(CS_CMD_ACTOR_CUE_599),
+};
+
+#undef ENUM_TO_STRING
+
+// see
+// https://github.com/zeldaret/mm/blob/0c7b90cf97f26483c8b6a98ae099a295f61e72ab/include/z64cutscene.h#L109-118
+const std::string csMMModifySeqTypes[] = {
```

Comment:
Unless I missed something these are only accessed with `.c_str()` so they can just be arrays of c strings.

Reply:
this file has been deleted since (replaced by xml magic) though I get bad results when using ``char*`` inside the map from ``CutsceneEnumData``, the output is wrong and idk why so I just used c_str instead, let me know if there's a fix or not (I forgot a lot about C++ over the years lol)